.github/pull_request_template.md

```diff
@@ -6,20 +6,20 @@ Fixes #[insert GH or internal issue number(s)].
 
 - Tests
   - [ ] TODO (before merge)
-  - [ ] Included
-  - [ ] Not necessary because:
+  - [ ] Tests included
+  - [ ] Tests not necessary because:
 - E2E Tests CI Job required? (Use "e2e" label or ask maintainer to run separately)
   - [ ] I don't know
   - [ ] Required / Maybe required
```

Comment:
nit: I've found `Maybe required` to be rather confusing. IMO "maybe" requires == me not knowing if e2es are required or not. So perhaps to avoid any confusion we can have:
```
- [ ] I don't know
- [ ] Required
- [ ] Not required because:
```